packages/components/psammead-media-player/src/Guidance/index.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { string, node } from 'prop-types';
+import styled from 'styled-components';
+import { C_WHITE } from '@bbc/psammead-styles/colours';
+import { mediaIcons } from '@bbc/psammead-assets/svgs';
+import {
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+  GEL_SPACING_HLF,
+} from '@bbc/gel-foundations/spacings';
+import { GEL_LONG_PRIMER } from '@bbc/gel-foundations/typography';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+
+const GUIDANCE_BACKGROUND = 'rgba(34, 34, 34, 0.75)';
+
+const StyledGuidance = styled.div`
+  ${({ service }) => getSansRegular(service)}
+  ${GEL_LONG_PRIMER};
+`;
+
+const GuidanceWrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  background-color: ${GUIDANCE_BACKGROUND};
+  color: ${C_WHITE};
+`;
+
+const Content = styled.div`
+  display: flex;
+  align-items: center;
+  padding: ${GEL_SPACING_DBL};
+`;
+
+const GuidanceMessage = styled.strong`
+  padding--left: ${GEL_SPACING_HLF};
```

Comment:
Does this work? I would expect this to have caused an invalid property warning in dev-tools and a visual regression.

Reply:
@simonsinclair  typo, fixed now